src/components/SingleDeviceGame/SingleDeviceGame.js

```diff
@@ -35,13 +43,22 @@ function SingleDeviceGame({players, spies, openSettings}) {
     }
 
     // Update the options state with the new array
-    setOptions(optionsCopy);
-    getPlace(i18n.language).then(res => 
-      setPlace(res)
-    ); 
+    setOptions(optionsCopy);    
+    if (addedPlaces.length > 0) {

```

Comment:
.length > 0 has a very common usage.
Using util function isEmptyArray is very good practice